.github/workflows/ci.yml

```diff
@@ -39,6 +39,17 @@ jobs:
         run: |
           nextflow run ${GITHUB_WORKSPACE} -profile test,docker --outdir ./results
 
+      - name: Upload logs
+        uses: actions/upload-artifact@v3
+        if: success() || failure()
+        with:
+          name: default_run_logs
+          path: |
+            *.log
+            *.err
+            work/
+            results/
+
```

Comment:
is this in the TEMPLATE?

Reply:
Was going to ask exactly the same 😆

Reply:
I was more thinking add to template

Reply:
Ok, forget I said anything and remove if not necessary anymore then